specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -60,47 +59,45 @@ image:Platform_Spec-10.png[image]
 The EJB, servlet, application client, and
```

Comment:
These specs need renaming to `Jakarta Enterprtise Beans, Jakarta Servlet, application client and Jakarta Connectors`

Reply:
done